bin/reportdaily.py

```diff
@@ -56,6 +64,142 @@ class MissingSubCommand(ValueError):
 log.addHandler(logging.NullHandler())
 
 
+def cmd_init(args):
+    """Creates an initial config file for a user config file"""
+
+    # create config by hand
+    check_if_config_exists(args)
+
+    log.debug("INIT selected %s", args)
+    print("Init selected", args)
+    return 0
+
+
+def create_config(args):
+    """This function will create a config file where the user data is stored"""
+    # time to ask the user for his data
+
+    # name
+    if args.name == None:
+        print("Please enter your full name --> example: 'Max Musterman'")
+        name = input("Ihr name ist ? ")
+    else:
+        name = args.name
+    # team
+    if args.team == None:
+        team = input("Please enter your team name: ")
+    else:
+        team = args.team
+    # year
+    if args.year == None:
+        year = int(input("What year of training are you in? "))
+    else:
+        year = args.year
+    # time
+    today_date = date.today()
+    print(
+        f"You are ready to go.  Entries for this {today_date} can  bee added now with reportdaily add")
```

Comment:
* If you check for `None`, don't use the `==` operator. Use `is`.
* Don't use "Please" in your text. It just makes the whole string longer.
* In this case, you don't necessarily need the else part.

If I combine all these issues above, I would suggest:

```suggestion
    # name
    name = args.name
    if name is None:
        name = input("Enter your full name (firstname and surename): ")

    # team
    team = args.team
    if team is None:
        team = input("Please enter your team name: ")
        
    # year
    year = args.year
    if year == None:
        year = int(input("What year of training are you in? "))
        
    # time
    today_date = date.today()
    print(
        f"You are ready to go.  Entries for this {today_date} can  bee added now with reportdaily add")
```